src/components/message/MessageContent/message-content.component.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Content from '../../content/content';
+import MessageContentStyle from './message-content.style';
+
+const MessageContent = ({
+  as, title, transparent, children
+}) => {
+  return (
+    <MessageContentStyle type={ as } transparent={ transparent }>
```

Comment:
We'll want this component to have a `data-component` attr., as per the original implementation. 

Our automation suites use them to locate nodes on the DOM.

https://github.com/Sage/carbon/blob/bb334d269d099c93955a4e931982d6f183a1726d/src/components/message/message.js#L142